pkg/service/backup/backupspec/versioning.go

```diff
@@ -117,13 +116,5 @@ func ListVersionedFiles(ctx context.Context, client *scyllaclient.Client, snapsh
 		}
 	}
 
-	if len(versionedFiles) > 0 {
```

Comment:
Why you drop this logging ?
Please bring it back (unless it logged somewhere else and this is just redundant logging).

Reply:
I didn't like passing logger as function argument, so I decided to log it separately.
The logging was done only in `--restore-tables`, but I added a commit that introduces it in `--restore-schema` as well.
```go
	w.versionedFiles, err = ListVersionedFiles(ctx, w.client, w.run.SnapshotTag, hosts[0], srcDir)
	if err != nil {
		return errors.Wrap(err, "initialize versioned SSTables")
	}
	if len(w.versionedFiles) > 0 {
		w.logger.Info(ctx, "Chosen versioned SSTables",
			"dir", srcDir,
			"versioned_files", w.versionedFiles,
		)
	}
```